files/nginx.conf

```diff
@@ -18,8 +18,19 @@ http {
   # this is necessary for us to be able to disable request buffering in all cases
   proxy_http_version 1.1;
 
+  lua_shared_dict token_dict 5m;
```

Comment:
Minor: 5m means a shared dict of 5MB size, right?
Isn't that a bit excessive?

Reply:
Yes it can be fine tuned. 1m should be more than enough, 5m doesn't hurt.